ui/ui-frontend-common/src/app/modules/components/header/menu/menu.component.html

```diff
@@ -0,0 +1,46 @@
+<div 
+    class="container p-0"
+    [@opacityAnimation]="state"
+    >
+    <div class="px-3 d-flex justify-content-end align-items-center header">
+        <vitamui-common-search-bar
+            class="col-6 p-0 mr-auto"
+            name="category-search"
+            (searchChanged)="onSearch($event)"
+            (clear)="resetSearch()"
+            placeholder="Saisir le nom de l'application" 
+            i18n-placeholder="@@categorySearchPlaceholder"
+        ></vitamui-common-search-bar>
+        <span class="pr-2" i18n="menuComponent@@myApplications">Mes applications</span>
+        <button (click)="onClose()" mat-icon-button class="apps-button">
+            <i class="vitamui-icon vitamui-icon-apps-colored"><span class="path1"></span><span class="path2"></span><span class="path3"></span><span class="path4"></span><span class="path5"></span><span class="path6"></span><span class="path7"></span><span class="path8"></span><span class="path9"></span></i>
```

Comment:
C'est quoi ces classes de path ? Que porte t-elles ?

A quoi servent ells ? Et pourquoi en initialiser autant (en particulier pour MANUAL_INGEST_APP où on en initialiser 188 ?)
Si c'est pour lister des éléments pourquoi ne pas utiliser `*ngFor in` ?

Reply:
Les classes de path sont générées automatiquement par la librairie Icomoon lors de l'ajout d'une icone complexe, comme c'est le cas pour les nouvelles icones des applications ^^ L'icône globale est composée de multiples entitées traduites chacune comme des "path", comme si c'était une grosse icone composée de sous icônes. Les paths sont déclarés dans le fichier des classes icomoon vitamui-icons.css